serverless/tensorflow/faster_rcnn_inception_v2_coco_gpu/nuclio/function.yaml

```diff
@@ -0,0 +1,134 @@
+metadata:
```

Comment:
The biggest problem which I see here is code duplication. One way to solve that is to use nuctl arguments: 

```bash
nuctl deploy tf-faster-rcnn-inception-v2-coco-gpu \
    --project-name cvat --path . --platform local \
    --base-image tensorflow/tensorflow:2.1.1-gpu \
    --desc "Faster RCNN from Tensorflow Object Detection GPU API" \
    --image cvat/tf.faster_rcnn_inception_v2_coco_gpu \
    --resource-limit nvidia.com/gpu=1
```

But it isn't possible to override meta.name. I don't see a full solution for now. Does the command above is good enough? I don't believe that somebody will decide to install both CPU and GPU versions at the same time. In any case they will get two functions with the same display names but different descriptions.

Reply:
I agree there is code duplication. I was going to discuss this here cause I wasn't sure if people want to have cpu and gpu at the same time.

runs on GPU
```
nuctl deploy tf-faster-rcnn-inception-v2-coco-gpu \
    --project-name cvat --path . --platform local \
    --base-image tensorflow/tensorflow:2.1.1-gpu \
    --desc "Faster RCNN from Tensorflow Object Detection GPU API" \
    --image cvat/tf.faster_rcnn_inception_v2_coco_gpu \
    --resource-limit nvidia.com/gpu=1
```
runs on cpu:
```
nuctl deploy tf-faster-rcnn-inception-v2-coco-gpu \
    --project-name cvat --path . --platform local \
    --base-image tensorflow/tensorflow:2.1.1-gpu \
    --desc "Faster RCNN from Tensorflow Object Detection GPU API" \
    --image cvat/tf.faster_rcnn_inception_v2_coco_gpu
```

This will fix duplication but then we have to be careful of the number of GPU functions we deploy. Any gpu function we deploy is going to be loaded during initialization and takes some GPU memory even if we don't use them.

PS, I'm not sure if we run the CPU version of top, if the speed would make any difference. Cause in that case tensorflow-GPU is installed, there might be some overhead checking for Cuda and see it doesn't exist but that should be very negligible in theory. Should I apply the changes and remove the code duplications?

Reply:
Let's make a note inside documentation about the limitation. Need to think how to solve the problem in the future. I hope nuclio should help us somehow here.

Reply:
- [ ] Remove duplicated code and use arguments for gpu
- [ ] Improve documentation for gpu limitation